.tool-versions

```diff
@@ -1,4 +1,4 @@
-erlang 25.2.1
+erlang 25.3.2
```

Comment:
Why not use the latest erlang version: `25.3.2.2`?

Reply:
a) I was considering a future contribution to the ci which includes parsing the .tool-versions file for elixir/otp versions and I previously had issues with using the extra erlang patch version when trying to use the tool versions in ci.
```
        uses: erlef/setup-beam@v1
        with:
          version-file: .tool-versions
          version-type: strict
``` 

b) I didn't want to make multiple PRs to bring the extension repo inline as well.

Reply:
ok